components/brave_wallet_ui/page/container.tsx

```diff
@@ -272,41 +272,41 @@ function Container (props: Props) {
   // This will scrape all of the user's accounts and combine the asset balances for a single asset
   const fullAssetBalance = (asset: ERCToken) => {
```

Comment:
The typing feels a bit odd to me. It could either be a `Number` or a string (`''`). However when we use it, we do `fullAssetBalance(asset)?.toString()`, so I assume `undefined` is also a possibility. For such functions, I think an optional `Number` would be a good return type.

Also let's be more conservative with use of empty string (`''`) in functions that are not rendered directly in components. `undefined` would be a better choice there.

Reply:
Added return types!